src/herbie.rkt

```diff
@@ -84,6 +99,7 @@
     [("--profile") "Whether to profile each run"
      (set! report-profile? true)]
     #:args (input output)
+    (operator-pruning)
```

Comment:
This sort of repetitiveness is probably a sign that we're not doing this the right way. But in this case I don't see an obvious better way. So I guess leave it as is.